CHANGELOG.md

```diff
@@ -7,14 +7,20 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 
-- introduces support for embedded messaging: an eligibility–based, personalized messages sent from Iterable to your mobile and web apps, which can display them inline, using native interface components
+- Introduces support for embedded messaging: an eligibility–based, personalized messages sent from Iterable to your mobile and web apps, which can display them inline, using native interface components
```

Comment:
It mentions "web apps", but I guess it's just trying to describe the feature as a whole, so this is probably fine.

Reply:
Yes thats what I felt too.. 👍🏼